registration/forms.py

```diff
@@ -95,6 +95,11 @@ def clean_email(self):
 
 def register_form_clean(self):
     cleaned_data = super(self.__class__, self).clean()
+    organ_choicese = [value for key, value in cleaned_data.items()
+                      if key.startswith('include') or key.startswith('allow')]
+    if organ_choicese:
+        if not any(organ_choicese):
+            raise django.forms.ValidationError("Please select  at least  one option to proceed ")
```

Comment:
@mabuaisha You have extra spaces in the error message, also, I think it will be better to set the message initially to "At least one organ/tissue need to be selected"